v0/destinations/pardot/data/PardotIdentify.json

```diff
@@ -0,0 +1,277 @@
+[
+    {
+        "destKey": "email",
+        "sourceKeys": [
+      "traits.email",
+      "context.traits.email"
+        ],
+        "required": true
+      },
+      {
+        "destKey": "campaign_id",
+        "sourceKeys": [
+      "campaignId",
+      "traits.campaignId",
+      "traits.campaign_id",
+      "context.traits.campaignId",
+      "context.traits.campaign_id"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "salutation",
+        "sourceKeys": [
+      "traits.salutation",
+      "context.traits.salutation"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "first_name",
+        "sourceKeys": [
+      "traits.first_name",
+      "context.traits.first_name",
+      "traits.firstName",
+      "context.traits.firstName"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "last_name",
+        "sourceKeys": [
+      "traits.last_name",
+      "context.traits.last_name",
+      "traits.lastName",
+      "context.traits.lastName"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "password",
+        "sourceKeys": [
+      "traits.password",
+      "context.traits.password"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "company",
+        "sourceKeys": [
+      "traits.company",
+      "context.traits.company"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "prospect_account_id",
+        "sourceKeys": [
+            "prospect_account_id",
+            "traits.prospect_account_id",
+            "traits.prospectAccountId",
+            "context.traits.prospect_account_id",
+            "context.traits.prospectAccountId"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "website",
+        "sourceKeys": [
+            "traits.website",
+            "context.traits.website"   
+        ],
+        "required": false
+      },
+      {
+        "destKey": "notes",
+        "sourceKeys": [
+            "traits.notes",
+            "context.traits.notes"   
+        ],
+        "required": false
+      },
+      {
+        "destKey": "job_title",
+        "sourceKeys": [
+            "traits.job_title",
+            "traits.jobTitle",
+            "context.traits.job_title",
+            "context.traits.jobTitle"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "department",
+        "sourceKeys": [
+            "traits.department",
+            "context.traits.department"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "country",
+        "sourceKeys": [
+            "traits.country",
```

Comment:
Let's add `traits.address.country` and `context.traits.address.country` as well with highest precendence

Reply:
added